Sources/BraveUI/SwiftUI/CustomPrivacySensitiveModifier.swift

```diff
@@ -17,7 +17,11 @@ private struct CustomPrivacySensitiveModifier: ViewModifier {
     Group {
       if isBackgrounded || isCaptured {
         content
-          .redacted(reason: .placeholder)
+          .opacity(0)
```

Comment:
Could we keep the .redacted here, but utilize the [privacy](https://developer.apple.com/documentation/swiftui/redactionreasons/privacy) reason instead? Seems like that would keep a standardized styling to address the issue.

The thing I'm not certain about is whether or not the use of .redacted versus .opacity might allow the content to be considered non sensitive and then still be accessible in some odd way.

Reply:
`.redacted(reason: .privacy)` doesn't work here for some reason. A comment in this file is probably related to this:
```
/// Custom modifier to behave similar to `.privacySensitive()`
/// The `.privacySensitive()` doesn't work in sheets because the
/// `redactedReasons` doesn't get passed through the environment in sheets.
```
Anyway, under the hood it works the same way, we apply the rule when `isBackgrounded || isCaptured` condition is met, so with `opacity(0)` we make sure the content is never visible.